static/elements/chromedash-myfeatures.js

```diff
@@ -78,9 +78,12 @@ class ChromedashMyFeatures extends LitElement {
     `;
   }
 
-  renderPendingApprovals() {
-    return this.renderBox(
+  renderPendingAndRecentApprovals() {
+    const pendingBox = this.renderBox(
       'Features pending my approval', 'pending-approval-by:me', 'approvals');
+    const recentBox = this.renderBox(
+      'Recently reviewed features', 'is:recently-reviewed', 'approvals', false);
+    return [pendingBox, recentBox];
```

Comment:
What does it mean to return a list of html here? Does it get rendered side by side?

Reply:
Not side-by-side, just one string of HTML after the other.   The resulting page looks like this:

![image](https://user-images.githubusercontent.com/17365/141033016-e71e579f-7b5f-4bb0-a32b-085f70c49a16.png)